### PR TITLE
New: `el : (string, Attributes, HTMLElement[]) -> HTMLElement`

### PR DESCRIPTION
**Pull request by joshcox**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,12 @@
   "name": "elsx",
   "description": "General Dom Object Generator; Possible JSX compilation target",
   "version": "0.0.0",
-  "keywords": ["dom", "jsx", "createElement"],
+  "keywords": [
+    "dom",
+    "jsx",
+    "createElement",
+    "el"
```
Comment (johnfoley3) [inline package.json:9]: ❓ Is `el` a larger concept, similar to `o` is for a `compose` function? The only thing I know of the `keywords` field of `package.json` is to aid in search indexing for packages, where a larger `el` keyword would come in handy.

**Review by johnfoley3 — APPROVED** [review]
Question on the `keywords`. Actual function looks good to me. I'm not sure exactly how to test it yet, but the code looks like it would work as I'd expect

File: package.json (hunk 1 above)
Comment (joshcox) [reply package.json:9]: It's a common shorthand name for dom-element generation functions.

**Comment by joshcox** [comment]
@johnfoley3, I added specs to the branch.

**Comment by johnfoley3** [comment]
Specs look great! Thank you for adding those.

**Pull request merged**